content/docs/0.16.x/bridge/_index.md

```diff
@@ -0,0 +1,248 @@
+        width="50%">}}
+  
+* For your convenience the SLOs that the evaluation was executed against, as well as the event payload, can be viewed.
+
+    {{< popup_image
+        link="./assets/services-view-slo.png"
+        caption="Evaluation SLOs"
+        width="50%">}}
+
+    {{< popup_image
+        link="./assets/services-view-payload.png"
+        caption="Event Payload"
+        width="50%">}}
+        
+* If you configured some remediations and one of them is currently running, the affected service, deployment and stage is indicated. You can also see, which type of remediation is
+  being executed and view its configuration and sequence execution.
+
+    {{< popup_image
+        link="./assets/services-view-remediations.png"
+        caption="Open remediations"
+        width="50%">}}
+
+    {{< popup_image
+        link="./assets/services-view-remediation-config.png"
+        caption="Remediation configuration"
+        width="50%">}}
+
+### Evaluation board
+
+* Next to the evaluation comparison as a heatmap or chart in the *Services* view, we provide a link to the *Evaluation Board*, which displays only 
+  the evaluation comparison on this stage. 
+  
+    {{< popup_image
+    link="./assets/bridge_evaluationboard.png"
+    caption="Keptn Bridge Evaluation Board">}}
+  
+### Sequences view
+
+* The sequences for the selected project are shown on this screen. It has a lot of filter possibilities, that can either be selected
+  in the left pane or be applied by using the search bar.
+* When you select a sequence, the related tasks are shown for each stage. The stages can be selected by clicking onto the stage name in
+  the timeline, or the badge of the sequence list item.
+* You can also expand each task to show the related events.
+
+  {{< popup_image
+  link="./assets/sequences-screen-filter.png"
+  caption="Keptn Bridge Sequences View">}}
+  
+### Uniform view
+
+The uniform view gives you an overview over all integrations installed within your Keptn installation. 
+The data is fetched from your Kubernetes cluster and shows useful information like
+* Unread/New error events
+* Deployment version of the service
+* The cluster or host where it is running
+* The Kubernetes namespace
+* The location within Keptn
+* The Cloud Events the services are subscribed to
+
+
+{{< popup_image
+link="./assets/uniform-view.png"
+caption="Keptn Bridge Uniform View">}}
+
+You can view the related subscriptions, events on error level, and sequence failures by selecting an integration. It is also possible to directly configure the subscriptions to listen to specific events.
+If there are new error events since the last time the integration was viewed, a red indicator shows up for this integration. When a sequence in this particular scope failed, then there is
+a link provided that takes you directly to the failed task in the sequence. 'n/a' in this regard means that the error is not related to any sequence execution. 
```

Comment:
```suggestion
If there are new error events since the last time the integration was viewed, a red indicator shows up for this integration. When a sequence in this particular scope fails,
a link is provided that takes you directly to the failed task in the sequence. 'n/a' in this regard means that the error is not related to any sequence execution. 
```